backend/bindings/execute.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 De-labtory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bindings
+
+// URL : /execute
+type ExecuteRequest struct {
+	Address          string
+	FunctionSelector string
+	Params           []Param
+}
+
+type Param struct {
```

Comment:
I think adding comments would be helpful. like type is for what, value is for what

Reply:
@zeroFruit 

Umm.. they are just type and value of parameter. What does need..?! Show me some examples!